### DISCUSSION TITLE
Mendeley Reference Manager und Stichwort 'usera' in Biblatex template

### DISCUSSION THREAD

**Discussion by ghost** [andygrunwald/FOM-LaTeX-Template#208] [Q&A]
Hallo liebes Entwicklerteam,

ich habe mich gewundert, weshalb in meiner Mendeley-Literaturverwaltungssoftware der Feldtyp 'usera' nicht auftaucht.

Nach ein wenig Durchsuchen des Leitfadens habe ich gesehen, dass nach dem(n) Verfasser(n) und vor der Jahreszahl ein Stichwort (ein bis zwei Wörter maximal) stehen muss (abh. vom Inhaltstyp, z. B. Monographie, Zeitschriftenartikel, etc.).

In der Tat kann man in Mendeley keinen neuen Inhaltstyp erstellen. Deshalb habe ich mich entschlossen innerhalb vom Mendeley den Inhaltstyp 'shorttitle' statt 'usera' zu nehmen. Dazu musste ich die Biblatex-Vorlage 'modsBiblatex2018.tex' abändern, in dem ich alle Einträge von 'usera' zu 'shorttitle' abänderte.

Nach der Eingabe des Stichworts in das Mendeley Attribut-Feld 'Short Title' wird nun die Fussnote richtig ergänzt.

Meine Bitte wäre, dass einer von Euch meine Anmerkung als Kommentar in die Vorlage einfügt, damit die nachfolgenden Generationen von Mendeley-Usern einen Benefit haben.

Bin Euch im Übrigen super dankbar für die tolle LaTeX-Vorlage. Wirklich sehr gelungen.

Beste Grüße 

Bojan

**Comment by andygrunwald** [comment]
Hey @basilius-darius,
Danke :) 
Hättest du ggf. Lust und Zeit den Kommentar als Pull Request vorzuschlagen? Das könntest du sogar über die UI machen:

1. Die entsprechende Datei unter https://github.com/andygrunwald/FOM-LaTeX-Template suchen
2. Auf die Datei klicken und oben rechts den Stift nutzen
3. Änderung durchführen + nach unten scrollen um eine Änderungsnachricht einzugeben (+ Create a new branch for this commit and start a pull request.-Aktivieren)

Das wäre Klasse :)
LG Andy
